### PR TITLE
update AccountCreatedPage towards styled components, and update reusa…

### PR DESCRIPTION
**Pull request by jay-hodgson**
…ble styled components

### CODE REVIEW

File: src/components/ProfileValidation/ProfileValidation.tsx [hunk 1]
```diff
@@ -2,9 +2,9 @@ import React, { useState, useEffect } from 'react'
 import { Redirect } from 'react-router-dom'
```
Comment (jay-hodgson) [inline src/components/ProfileValidation/ProfileValidation.tsx:2]: formatting

File: src/components/ProfileValidation/ProfileValidation.tsx [hunk 2]
```diff
@@ -373,25 +456,42 @@ export const ProfileValidation = (props: ProfileValidationProps) => {
...
-              <BodyControlFactory {...{
-                step: step, verificationSubmission: verificationSubmission, onFormChange: isFormComplete => {
-                  setIsContinueButtonEnabled(isFormComplete)
-                },
-                onReturnToSettings: () => {
-                  setIsReturnToAccountSettings(true)
-                }
-              }} />
+              <SourceAppLogo
+                sx={{ textAlign: 'center', paddingBottom: '35px' }}
```
Comment (jay-hodgson) [inline src/components/ProfileValidation/ProfileValidation.tsx:471]: Added paddingBottom, and removed margin value (was not being applied)

File: src/components/StyledComponents.ts [hunk 3]
```diff
@@ -1,24 +1,31 @@
...
-    minHeight: '100vh',
-    paddingTop: '100px',
-    background: '#d5dbe1',
+export const StyledOuterContainer = styled(Box, {
+  label: 'StyledOuterContainer',
+})(({ theme }) => ({
+  minHeight: '100vh',
+  paddingTop: '50px',
+  paddingBottom: '50px',
+  background:
```
Comment (jay-hodgson) [inline src/components/StyledComponents.ts:9]: Changed background

**Comment by jay-hodgson** [comment]
Screen Shot 2023-01-20 at 12 23 55 PM
Screen Shot 2023-01-20 at 12 24 35 PM

**Review by conundrumgirl — APPROVED** [review]

**Pull request merged**